AutomatedTesting/Gem/PythonTests/atom_renderer/atom_hydra_scripts/hydra_GPUTest_LightComponent.py

```diff
@@ -0,0 +1,275 @@
+        as well as the Global Skylight (IBL) component.
+    4. Enters game mode to take a screenshot for comparison, then exits game mode.
+    5. Selects the "ground_plane" entity and changes updates the material to a new material.
+    6. Enters game mode to take a screenshot for comparison, then exits game mode.
+    7. Selects the "spot_light" entity and increases the Light component Intensity to 800 lm
+    8. Enters game mode to take a screenshot for comparison, then exits game mode.
+    9. Selects the "spot_light" entity and sets the Light component Color to 47, 75, 37
+    10. Enters game mode to take a screenshot for comparison, then exits game mode.
+    11. Selects the "spot_light" entity and modifies the Shutter controls to the following values:
+        - Enable shutters: True
+        - Inner Angle: 60.0
+        - Outer Angle: 75.0
+    12. Enters game mode to take a screenshot for comparison, then exits game mode.
+    13. Selects the "spot_light" entity and modifies the Shadow controls to the following values:
+        - Enable Shadow: True
+        - ShadowmapSize: 256
+    14. Modifies the world translate position of the "spot_light" entity to 0.7, -2.0, 1.9 (for casting shadows better)
+    15. Enters game mode to take a screenshot for comparison, then exits game mode.
+
+    Finally prints the string "Light component tests completed" after completion
+
+    Tests will fail immediately if any of these log lines are found:
+    1. Trace::Assert
+    2. Trace::Error
+    3. Traceback (most recent call last):
+
+    :return: None
+    """
+    atom_component_helper.create_basic_atom_level(level_name=LEVEL_NAME)
+
+    # Run tests.
+    area_light_test()
+    spot_light_test()
+    general.log("Light component tests completed.")
+
+
+def area_light_test():
+    """
+    Basic test for the "Light" component attached to an "area_light" entity.
+    Example: Entity addition/deletion, & enters game mode w/ Light component attached using various light types.
+    """
+    # Create an "area_light" entity with "Light" component using Light type of "Capsule"
+    area_light_entity_name = "area_light"
+    light_component = "Light"
+    area_light = hydra.Entity(area_light_entity_name)
+    area_light.create_entity(math.Vector3(-1.0, -2.0, 3.0), [light_component])
+    general.log(
+        f"{area_light_entity_name}_test: Component added to the entity: "
+        f"{hydra.has_components(area_light.id, [light_component])}")
+    light_component_id_pair = atom_component_helper.attach_component_to_entity(area_light.id, light_component)
+    light_type_property = 'Controller|Configuration|Light type'
+
+    # Select the "Capsule" light type option.
+    capsule_light_type = atom_component_helper.LIGHT_TYPES['capsule']
+    azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast,
+        'SetComponentProperty',
+        light_component_id_pair,
+        light_type_property,
+        capsule_light_type
```

Comment:
I feel that putting atom_component_helper.LIGHT_TYPES directly as an argument rather than creating the capsule_light_type variable would be more clear to me. the pattern is used for each light_type_property call... either way is fine.

Reply:
i agree, it was leftover copypasta and updating to use dict directly makes more sense